README.md

```diff
@@ -19,3 +19,11 @@ where:
 * AZURE_TENANT_ID
 
 In case you are using a GitHub resource in your overlay that is a private repository, mount your the ssh key that is authorized to access GitHub with -v PATH_TO_ID_RSA_FILE:/root/.ssh/id_rsa
+
+If you are using a full base+overlay dir structure you can use following command:
+
+```docker run ingrammicro/kustomize-sops:v1.0.0 -e AZURE_CLIENT_ID=xxxx -e AZURE_CLIENT_SECRET=xxx -e AZURE_TENANT_ID=xxxx -v KUSTOMIZE_BASE:/kust KUSTOMIZE_OVERLAY [-v ~/.ssh/id_rsa:/root/.ssh/id_rsa]```
```

Comment:
Being a entrypoint argument, KUSTOMIZE_OVERLAY needs to be moved to the end of the line, after the "-v" option

Reply:
Fixed